keep/api/models/db/statistics.py

```diff
@@ -0,0 +1,8 @@
+from sqlmodel import Field, SQLModel
+
+
+class PMIMatrix(SQLModel, table=True):
+    tenant_id: str = Field(foreign_key="tenant.id")
+    fingerprint_i: str = Field(primary_key=True)
+    fingerprint_j: str = Field(primary_key=True)
+    pmi: float
```

Comment:
Add empty line at the end of file
```suggestion
    pmi: float
    
```

Reply:
Thx! Addressed [here](https://github.com/keephq/keep/commit/76508eb1a91805d71e26669945b05157dbb96486)